app/views/turns/index.html.erb

```diff
@@ -0,0 +1,497 @@
+      }
+      
+      .cards {
+        display: grid;
+        justify-content: center;
+        grid-template-columns: repeat(3, 1fr);
+        grid-template-rows: 1fr;
+        gap: 5px;
+        margin-top: 20px;
+        margin: 10px auto;
+        padding: 0;
+      }
+    }
+
+  </style>
+</head> 
+<body>  
+  <div class="back">
+    <a href="#">< Back</a>
+  </div>
+  <header class="header">
+    <h2 class="turn__title">manage turns</h2>
+    <h3 class="turn__subtitle">turn list</h3>
+
+    <div class="wrapper">
+      <div class="search-input">
+        <a href="" target="_blank" hidden></a>
+        <input type="text" placeholder="Type to search..">
+        <div class="autocom-box">
+          <!-- here list are inserted from javascript -->
+        </div>
+        <div class="icon"><i class="fas fa-search"></i></div>
+      </div>
+    </div>
+  </header>
+
+  <main class="turns"> 
+    <div class="cards">
+
+      <div class="card">
+        <div class="card__cb">
+          <div class="card__content">
+            <p>monday 7:00am - 8:00am</p>
+            <span>manager #1 name</span>
+          </div>
+          <button class="btn">check</button>
+        </div>
+      </div>
+
+      <div class="card">
+        <div class="card__cb">
+          <div class="card__content">
+            <p>monday 7:00am - 8:00am</p>
+            <span>manager #1 name</span>
+          </div>
+          <button class="btn">check</button>
+        </div>
+      </div>
+
+      <div class="card">
+        <div class="card__cb">
+          <div class="card__content">
+            <p>monday 7:00am - 8:00am</p>
+            <span>manager #1 name</span>
+          </div>
+          <button class="btn">check</button>
+        </div>
+      </div>
+
+      <div class="card">
+        <div class="card__cb">
+          <div class="card__content">
+            <p>monday 7:00am - 8:00am</p>
+            <span>manager #1 name</span>
+          </div>
+          <button class="btn">check</button>
+        </div>
+      </div>
+
+      <div class="card">
+        <div class="card__cb">
+          <div class="card__content">
+            <p>monday 7:00am - 8:00am</p>
+            <span>manager #1 name</span>
+          </div>
+          <button class="btn">check</button>
+        </div>
+      </div>
+
+      <div class="card">
+        <div class="card__cb">
+          <div class="card__content">
+            <p>monday 7:00am - 8:00am</p>
+            <span>manager #1 name</span>
+          </div>
+          <button class="btn">check</button>
+        </div>
+      </div>
+
+    </div>
+  </main>
+
+  <div class="circle">
+    <i class="fas fa-plus icon__plus"></i>
+  </div>
+
+  <footer>
+    <div class="left">
+      <span class="bc">Brightcode</span>
+      <i class="fa fa-github-square"></i>
+    </div>
+
+    <div class="right">
+      <p class="rs">reileros 3</p>
+      <h4 class="nc">Nutricheck</h4>
+    </div>
+  </footer>
+  <script>
```

Comment:
Please move this code to a separate .js file

Reply:
Alright, I'm going to move the js code to its corresponding file.